angular/demo/bootstrap/src/app/samples/progressbar/fullCustom.route.ts

```diff
@@ -27,31 +27,31 @@ import CoffeeProgressbarComponent from './cofee-progressbar.component';
 					<button class="btn btn-outline-primary" [disabled]="!progressbar.state().started" (click)="stop(true)">Reset</button>
 				</div>
 				<p class="mt-3">
-					<span>{{ value === 0 ? 'Need to wake up.' : value < 100 ? 'Retrieving coffee... ' + value + '%' : 'Ready to work !' }}</span>
+					<span>{{ !subscription ? 'Need to wake up.' : value() < 100 ? 'Retrieving coffee... ' + value() + '%' : 'Ready to work !' }}</span>
```

Comment:
```suggestion
					<span>{{ value() === 0 ? 'Need to wake up.' : value() < 100 ? 'Retrieving coffee... ' + value() + '%' : 'Ready to work !' }}</span>
```

Reply:
The `!subscription` was added to to slightly improve the user experience.